pkg/ci/main.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+	"net/http"
+	"sort"
+	"text/template"
+	"time"
+
+	"github.com/alexellis/arkade/pkg/get"
+)
+
+func URLExists(name, url, version string) error {
+	fmt.Println("--------------->>>>>>>>>>>>>>>>>>>>>>>>>", name)
+	timeout := time.Second * 10
+	client := get.MakeHTTPClient(&timeout, false)
+
+	req, err := http.NewRequest(http.MethodHead, url, nil)
+	if err != nil {
+		return err
+	}
+
+	res, err := client.Do(req)
+	if err != nil {
+		return err
+	}
+
+	if res.Body != nil {
+		defer res.Body.Close()
+	}
+
+	if res.StatusCode != http.StatusOK {
+		return fmt.Errorf("tool %s with version %s not found", name, version)
+	}
+	return nil
+}
+
+func main() {
```

Comment:
Why did you pick a main and not Go's testing framework for this?

Reply:
@alexellis  I don't understand what you mean?

Reply:
Could you convert the main.go into a unit test file please?

Then use t.Testing etc.